.github/workflows/devbuild.yml

```diff
@@ -254,16 +254,28 @@ jobs:
         run: |
           export HOMEBREW_NO_AUTO_UPDATE=1
           export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
+          if [ "${{ matrix.os }}" == "macos-13" ] ; then
+            # GitHub Action macOS 13 has a problem with python3
+            brew list -1 | grep python | while read formula; do brew unlink $formula; brew link --overwrite $formula; done;
+          fi
           brew install qt6
 
       - name: dependency by pip
         run: |
           echo "which python3: $(which python3)"
           ls -al $(which python3)
-          python3 -m pip -v install --upgrade setuptools
-          python3 -m pip -v install --upgrade pip
-          python3 -m pip -v install --upgrade numpy pytest flake8
-          python3 -m pip -v install --upgrade pyside6==$(qmake -query QT_VERSION)
+          if [ "${{ matrix.os }}" == "macos-13" ] ; then
+            # suppress the warning of pip because of python link issue of brew
+            python3 -m pip -v install --upgrade setuptools  --break-system-packages
```

Comment:
unfortunately we need this `--break-system-packages` due to brew python link issue

Reply:
This is not something we should treat as normal, but not a horrible no for a workaround for an issue specific to one version on one platform.